### PR TITLE
edit words

### PR DESCRIPTION
**Pull request by jaybee84**
### Purpose

Reduce words to fit 6000 word limit #231 

### Directions for reviewers

I reduced 100-ish words without rewriting significantly. We are still over the 6000 word limit by 300 words. Do you see more opportunities to reduce words ?

#### Which areas should receive a particularly close look?

Body, Boxes (figure legends not included in the word limit)

#### Is the pull request ready for review?

Yes

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

**Comment by jaybee84** [comment]
wc- 6182

File: content/01.synopsis.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ## Abstract {.page_break_before}
 
-The advent of high-throughput profiling methods such as genomics and other technologies has accelerated basic research and made deep molecular characterization of patient samples routine. 
-These approaches provide a rich portrait of genes, molecular pathways, and cell types involved in disease phenotypes. 
+The advent of high-throughput profiling methods (such as genomics or imaging) has accelerated basic research and made deep molecular characterization of patient samples routine. 
+They provide a rich portrait of genes, molecular pathways, and cell types involved in disease phenotypes.
```
Comment (jaclyn-taroni) [inline content/01.synopsis.md]: This has swung too far in the other direction as to be somewhat unclear, in my opinion.
```suggestion
These approaches provide a rich portrait of genes, molecular pathways, and cell types involved in disease phenotypes. 
```

File: content/03.combining-datasets.md [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
+Steps like reprocessing data using a single pipeline, using batch correction methods [@doi:10.1093/biostatistics/kxj037; @doi:10.1093/nar/gku864], and normalizing raw values appropriately without affecting the underlying variance in the data [@doi:10.1186/gb-2010-11-3-r25, @doi:10.1371/journal.pcbi.1003531, @doi:10.1186/s13059-014-0550-8] may be necessary to mitigate unwanted variability. (Figure [@fig:1]a)
+Data harmonization may also entail standardization of sample labels using biomedical ontologies to normalize how samples are annotated across multiple datasets.
 
-Another way to improve the quality of a dataset is to improve the accuracy of metadata (both the description of technical variables and biologically relevant phenotypes) for each sample in the dataset. 
-This may increase the effectiveness of ML models in extracting biologically relevant patterns from small datasets. 
+Ways to improve the quality of a dataset include improving the accuracy of metadata (both the description of technical variables and biologically relevant phenotypes) for each sample in the dataset.
```
Comment (jaclyn-taroni) [inline content/03.combining-datasets.md]:
```suggestion
Improving the accuracy of metadata (both the description of technical variables and biologically relevant phenotypes) for each sample in the dataset can improve that dataset's quality.
```

File: content/05.model-complexity.md [hunk 3]
```diff
@@ -1,40 +1,36 @@
-## Reducing misinterpretation of model output with statistical techniques 
+## Reducing misinterpretation of model output with statistical techniques
 
-Machine learning methods generally work well on data that meet a few critical assumptions. 
-First, the dataset contains an equal number of samples for all categories (no “class imbalance”). 
-Second, the dataset is complete; all samples have measurements for all variables in the dataset (i.e., the dataset is not “sparse”, meaning that it is not missing data for some of the samples). 
+ML works well on data that meet a few critical assumptions.
```
Comment (jaclyn-taroni) [inline content/05.model-complexity.md]: If we can swing it...
```suggestion
ML generally works well on data that meet a few critical assumptions.
```

File: content/06.prior-knowledge.md [hunk 4]
```diff
@@ -1,31 +1,29 @@
...
 Effective methods to calculate node embeddings that can generate actionable insights for rare diseases is an active area of research [@doi:10.1142/9789811215636_0041].
 
 Another application of KGs is to augment or refine a dataset [@doi:10.1186/s12911-019-0752-9, doi:10.1186/s12911-019-0938-1].
-For example, Li et. al.[@doi:10.1186/s12911-019-0938-1] used a KG to identify linked terms in a medical corpus from a large number of patients, some with rare disease diagnoses.
-They were able to augment their text dataset by identifying related terms in the clinical text to map them to the same term - e.g., mapping "cancer" and "malignancy" in different patients to the same clinical concept. 
-With this augmented and improved dataset, they were able to train and test a variety of text classification algorithms to identify rare disease patients within their corpus. (Figure [@fig:4]b, Box 1a)
+Li et. al.[@doi:10.1186/s12911-019-0938-1] used a KG to identify linked terms in a medical corpus from a large number of patients, some with rare disease diagnoses.
```
Comment (jaclyn-taroni) [inline content/06.prior-knowledge.md]:
```suggestion
Li et. al. [@doi:10.1186/s12911-019-0938-1] used a KG to identify linked terms in a medical corpus from a large number of patients, some with rare disease diagnoses.
```

**Review by jaclyn-taroni — APPROVED** [review]
Looks good overall, leaving a few suggestions

**Pull request merged**